src/components/apiComponents/exampleSelector.jsx

```diff
@@ -0,0 +1,57 @@
+import { Select } from '@headlessui/react'
+import { ChevronDownIcon } from '@heroicons/react/20/solid'
+import clsx from 'clsx'
+
+const ApiExampleSelector = ({
+  examples,
+  selectedExample,
+  handleSetExample,
+}) => {
+  return (
+    <div className="w-full">
+      <label
+        className="mb-3 text-sm font-medium text-gray-800 dark:text-neutral-400"
+        htmlFor="endPoint"
+      >
+        Loaded Example
+      </label>
+      <div className="relative flex w-full gap-2">
+        <div className="relative flex h-12 w-full">
+          <Select
+            onChange={(e) => handleSetExample(e.target.value)}
```

Comment:
When a Loaded Example is set, should it set the endpoint to Solana mainnet (since that's where the examples are), and in the future if you add Eclipse loaded examples it could select Eclipse mainnet?

Reply:
I hadn't decided on this yet because we don't have aura mainnet ready for launch possibly with these examples. So we'll need to decide on how we are loading these. Either potentially with a custom with basic solana api endpoint. Or we let me people fill in their own and navigate.

This is why I ended up with the assets being in a group under "Solana Mainnet" to point this out.

Reply:
I haven't adjusted this at the moment incase people are using custom endpoints. Might be pretty annoying if everytime you select and example it swaps out your custom. Though I guess if "custom" then don't change maybe.

Reply:
i think it would be good to pick Solana mainnet if nothing is picked, but not change it if they have something picked, if that is easy to do.
Contingent on whether we are publishing the mainnet endpoint yet

Reply:
Added current behaviour:

If custom endpoint is selected and an example is selected DO NOT change the enpoint value from custom
else change endpoint value to match example.